docs/_writing-tests/server-features.md

```diff
@@ -94,4 +94,19 @@ of the response. For details see the
 [wptserve documentation](https://wptserve.readthedocs.org).
 
 
+### Writing tests for HTTP/2.0
+
+The server now has a prototype HTTP/2.0 server which gives you access to 
+some of the HTTP/2.0 specific functionality. Currently, the server is off
+by default and needs to be run using `./wpt serve --h2` in order to enable it.
+To find out more about the new Python handlers introduced head over to 
```

Comment:
"head over to" seems a little colloquial. I would say something like "The HTTP 2 server supports handlers that work per-frame; these are documented in […]".